src/Pangraph/Examples/ToContainersGraph.hs

```diff
@@ -0,0 +1,18 @@
+module Pangraph.Examples.ToContainersGraph where
```

Comment:
I think the example will be more useful if you used a small concrete graph, explaining the input and output in the comments (instead of reading from some file).